doc/02.1-Concepts-UNIX.md

```diff
@@ -0,0 +1,281 @@
+
+Wrapping up, **hardware + kernel + shell = usable computer**.
+As a user, the only part you are dealing with is the shell.
+Bash is your shell, in Bash you feel at home.
+
+In your virtual machines, you also have a **graphical shell** available to you, this is what you are interacting with when you boot it up and start clicking around.
+You can still fire up a **terminal** and access bash through the graphical interface.
+Even though most of our actual work will be done in a non-graphical environment, running a graphical shell is desirable over a 'pure terminal' environment, because this way you can have multiple terminals open on your screen, copy paste with a mouse, search things in a browser window etc.
+Note that while being optional in our class, ability to comfortably work in a purely non-graphical environment is an extremely valuable skill.
+
+## What is a distribution ?
+
+Sometimes a community (of software developers) will package a collection of software based on the Linux kernel, providing a complete and coherent operating system and **repositories of software**.
+Such collections are called 'distributions', and many popular ones are available for free.
+Examples are Debian, Ubuntu, Suse, RedHat, CentOS, Arch, and Mint.
+The Linux Mint we have running in our VMs are Ubuntu based, which is in turn Debian based.
+But note that the differences between distributions **will NOT be pronounced in your daily usage**, they are mostly evident in system administration and will be invisible to you as a user.
+Remember, all you are using is the shell, which is bash, which is POSIX compliant, be aware that there are different flavors of Linux based OS, but do not worry about the distribution unless you are also maintaining the machine.
+
+Note that we often use 'Linux' to refer to such a distribution as a complete OS and not the 'Linux kernel'.
+This is a conflicting but widely used terminology, and now you know what is what.
+
+## The super user
+
+As mentioned before, the POSIX standards are based on a multi-user scheme.
+Most of these are regular users which each belong to a set of groups, with given permissions, but one, namely **root**, is special in that it has unrestricted access to everything.
+**The user 'root' is the superuser, he does not have permissions, he sets the permissions.**
+
+In most occasions, you will be a regular user, and your scope will be limited by set **permissions**.
+In the case of our VM, you do have root access (password is i519) to your VM.
+You can prefix your commands with `sudo` to act as the superuser, [this](https://www.xkcd.com/149/) is pretty much how it works.
+In the rare occasion when you are the super user, you must tread carefully, for with great power comes great responsibility.
+As it is very easy to revert your VM to its initial state (re-download), feel free to try how much damage you can do with a simple command.
+
+### Excercise : Destroy your OS
+
+Backup your VM.
+
+`sudo rm -r /`
```

Comment:
This may make sense in the context of a class where specific instructions regarding VMs have been provided, but we wouldn't want somebody in our group running `sudo rm -r /` on gremlin2 or gremlin4 for the sake of learning a lesson!